### DISCUSSION TITLE
SES list management and api

### DISCUSSION THREAD

**Discussion by hamlethakobyan-zema** [aws/aws-sdk-net#3554] [Q&A] [answered]
Hi guys, [List management overview](https://docs.aws.amazon.com/ses/latest/dg/sending-email-list-management.html#list-management-overview) states that `Only one contact list is allowed per AWS account.` but we have method `ListContactLists` in SES API v2. Why we have such an inconsistency, or, maybe, I missed something?

**Comment by dscpinheiro** [comment, marked as answer]
We (the AWS SDK for .NET team) don't control how the APIs are designed (it's up to each service team), but it's possible they added a List operation so that if the limit is relaxed in the future existing customers won't need to update their SDK dependency.

You can use the [GetContactList](https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_GetContactList.html) method instead to retrieve the current list allowed in your account.
